app/tests/unit/components/utils.spec.js

```diff
@@ -10,6 +12,69 @@ helper.logHelper();
 
 const mockAxios = new MockAdapter(axios);
 
+const webadeList = require('./fixtures/webadeList.json');
+const webadeListWithInsecurePrefs = require('./fixtures/webadeListWithInsecurePrefs.json');
+
+describe('filterForInsecurePrefs', () => {
+  it('should return a list of webade configs with insecure prefs', async () => {
+
+    const result = await utils.filterForInsecurePrefs(webadeListWithInsecurePrefs, 'password|secret');
+
+    expect(result).toBeTruthy();
+    expect(result).toHaveLength(2);
+    expect(result[0].applicationAcronym).toEqual('ALTERNATE_API');
+    expect(result[1].applicationAcronym).toEqual('TEST_APP');
+    expect(result[0].preferences).toHaveLength(2);
+    expect(result[1].preferences).toHaveLength(1);
+  });
+
+  it('should not return a list of webade configs when there are 0 insecure prefs', async () => {
+    const result = await utils.filterForInsecurePrefs(webadeList, 'password|secret');
+    expect(result).toHaveLength(0);
+  });
+
+  it('should exception on an empty list', () => {
+    expect(() => { utils.filterForInsecurePrefs([], 'password|secret'); }).toThrow(Error);
+  });
+});
+
+describe('filterWebAdeDependencies', () => {
+  it('should return a list of webade config dependencies', async () => {
+
+    const result = await utils.filterWebAdeDependencies(webadeList, 'DMS');
+
+    expect(result).toBeTruthy();
+    expect(result).toHaveLength(1);
+    expect(result[0].applicationAcronym).toEqual('EXAMPLE2_API');
+  });
+
+  it('should throw on bad input', async () => {
+    expect(() => { utils.filterWebAdeDependencies(undefined, 'DMS'); }).toThrow(Error);
```

Comment:
As this is a one-liner intent, you probably don't need the surrounding {} braces and ending semicolon for this arrow function definition.

Reply:
Indeed